truetype/truetype.go

```diff
@@ -111,6 +120,8 @@ type Font struct {
 	bounds                  Bounds
 	// Values from the maxp section.
 	maxTwilightPoints, maxStorage, maxFunctionDefs, maxStackElements uint16
+	// First value from head table
+	nameRecord *nr
```

Comment:
*nr due to habit.

Eagerly reading them isn't necessary, no.  `parseHead` could be called from `Name` and `Style` if nr is nil (in the case of *nr), fontInfo contains only zero values, or if fontInfo is defined as a `[]string` and is nil.